### ISSUE TITLE
Intermittent failure on Basic low-res ScalarField localization

### ISSUE THREAD

**Issue by Affie** [JuliaRobotics/RoME.jl#626]

**Comment by dehann** [comment]
Yes, this test is numerically sensitive and still work in progress.  We can separate these tests out like IIF and then rerun just the most sensitive ones.  I added these tests as a 90% complete placeholder.  WIll get back to these after IIF 1010 is complete.
